terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -x
+exec > >(tee /tmp/user-data.log|logger -t user-data ) 2>&1
+
+#SETTING UP HOSTNAME
+
+REGION="${region}"
+IP=`curl -s http://169.254.169.254/latest/meta-data/public-ipv4`
+echo "aws-$REGION-$IP" > /etc/hostname
+hostname "aws-$REGION-$IP"
+
+#SETUP DATADOG
+apt-get update
+apt-get install awscli -y
+
+DATADOG_API_KEY_ENCRYPTED="${datadog_api_key}"
```

Comment:
I thought we'd fetch this secret from KMS in this script not passing as TF variable

Reply:
it's encrypted string, that instance is decrypting on boot

Reply:
yes, but I think that the instance should fetch the encrypted script too, not TF.

Reply:
done with usage of AWS Secrets Manager. Only epoch spot nodes can read this secrets permissions granted by IAM profile attache do node.